client/signup/config/flows-pure.js

```diff
@@ -118,6 +118,14 @@ export function generateFlows( {
 			lastModified: '2020-12-10',
 			showRecaptcha: true,
 		},
+		{
+			name: 'onboarding-with-email',
+			steps: [ 'user', 'domains', 'emails', 'plans' ],
+			destination: getSignupDestination,
+			description: 'Copy of the onboarding flow that includes an email step',
```

Comment:
I wonder if the following changes wouldn't help reduce the confusion around the reasons for having that new flow: 
```suggestion
			description: "Copy of the 'onboarding' flow that always includes an email step (this flow is used by the Professional Email landing page)",
```